packages/ecommerce-app-base/src/AppConfig/AppConfig.spec.tsx

```diff
@@ -62,6 +62,7 @@ const renderComponent = (sdk: unknown) => {
       logo="some-logo.svg"
       color="red"
       description="App description"
+      isInOrchestrationEAP={isInOrchestrationEAP ?? true}
```

Comment:
Why is the default true? Isn't Orchestration opt-in which would mean the default being false would be a more accurate representation?

Reply:
👍 I will apply this change in the [commercetools PR](https://github.com/contentful/apps/pull/3855).